src/main/java/uk/gov/hmcts/dts/fact/repositories/CourtAreaOfLawRepository.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.dts.fact.repositories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import uk.gov.hmcts.dts.fact.entity.CourtAreaOfLaw;
+
+public interface CourtAreaOfLawRepository extends JpaRepository<CourtAreaOfLaw, Integer> {
+    void deleteCourtAreaOfLawByCourtId(Integer id);
```

Comment:
This is a new method, I wonder you should add an integration test for this one

Reply:
Written one now :)